docs/source/10-minutes-to-dask.rst

```diff
@@ -75,30 +86,34 @@ including information about how the chunks should be structured.
 
    .. group-tab:: Array
 
-      .. code-block:: python
+      See :doc:`array`.
+
+      .. jupyter-execute::
```

Comment:
I really like the widget that pops up for Dask arrays, so that's why I changed this only for the Dask array sections.

Reply:
Oh I'm happy for you to change it for all of them if you like! I don't think I knew about `jupyter-execute`

Reply:
oh I assumed that the dataframe would also have their jupyter repr?

Reply:
it does! to me it didn't look all that different, but I can add it in.

Reply:
ok just pushed new changes w/ `jupyter-execute` added for dask bag and dask dataframe. visually, not sure which way I prefer (there's a bit more whitespace around some blocks, e.g.)

Reply:
hmm yeah I see what you mean. I am fine with either. I'll just merge whatever is here before releasing tomorrow :)

Reply:
ok, changed it back! thx for your help :)